packages/veritone-json-schemas/schemas/vtn-standard/transcript/transcript.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://docs.veritone.com/schemas/vtn-standard/transcript.json",
+  "title": "vtn-standard.object",
+  "description": "Standard engine output for Object at Veritone",
+  "type": "object",
+  "definitions": {
+    "words": {
+      "type": "array",
+      "minItems": 1,
+      "if": {
+        "minItems": 2
+      },
+      "then": {
+        "items": {
+          "type": "object",
+          "properties": {
+            "word": {
+              "$ref": "master.json#/definitions/word"
+            },
+            "confidence": {
+              "$ref": "master.json#/definitions/confidence"
+            },
+            "utteranceLength": {
+              "$ref": "master.json#/definitions/utteranceLength"
+            },
+            "bestPath": {
+              "type": "boolean"
+            }
+          },
+          "required": [
+            "word",
+            "confidence",
+            "utteranceLength"
+          ]
+        },
+        "uniqueItems": true,
+        "requireBestPath": true
+      },
+      "else": {
+        "items": {
+          "type": "object",
+          "properties": {
+            "word": {
+              "$ref": "master.json#/definitions/word"
+            },
+            "confidence": {
+              "$ref": "master.json#/definitions/confidence"
+            }
+          },
+          "required": [
+            "word"
+          ]
+        }
+      }
+    }
+  },
+  "properties": {
+    "validationContracts": {
+      "type": "array",
+      "items": {
+        "const": "transcript"
+      }
+    },
+    "series": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "words": {
+            "$ref": "#/definitions/words"
+          },
+          "startTimeMs": {
+            "type": "integer",
+            "minimum": 0
+          },
+          "stopTimeMs": {
+            "type": "integer",
+            "minimum": 0
+          },
+          "language": {
+            "$ref": "master.json#/definitions/language"
+          }
+        },
+        "required": [
+          "startTimeMs",
+          "stopTimeMs"
+        ]
+      }
+    },
+    "language": {
+      "$ref": "master.json#/definitions/language"
+    },
+    "schemaId": {
```

Comment:
Why does `transcript` define all these header fields but `object` doesn't?

Reply:
We pivoted from object, so I never got to finish it. We'll revisit it later with more tests and such.